src/video.rs

```diff
@@ -0,0 +1,37 @@
+use rusty_asm::rusty_asm;
+
+pub fn fill_screen(c: u8) {

```

Comment:
This will work in mode 13h, but not in other modes.    I wonder if some abstraction is needed somehow ?

Reply:
@stuaxo that's a good point, might be ideal to have a way of deferring to different mode-based fills down the line. i'm going to look into some other video modes, so maybe once i create some other graphical routines i can find a way to wrap them